dts/bindings/i2c/st,stm32-i2c-v2.yaml

```diff
@@ -13,3 +13,13 @@ properties:
 
     interrupts:
       required: true
+    # Precise timings values for a given Hardware can be pre-computed with a tool like STM32CubeMX
+    # or directly from I2C_TIMINGR register description.
+    # Because timing value is valid for a given I2C peripheral clock frequency and target I2C bus speed,
+    # each timing value must be provided with the matching configuration.
+    # The resulting table entries should look like <periph_clock i2c_speed_hz timing>
+    # # Example: <64000000 I2C_BITRATE_STANDARD 0x10707DBC>, <64000000 I2C_BITRATE_FAST 0x00603D56>, <56000000 I2C_BITRATE_STANDARD 0x10606DA4>, <56000000 I2C_BITRATE_FAST 0x00501D63>
```

Comment:
This details should be made part of the property description.

Reply:
I haven't found a description that is longer than a small line.
Should I put everything inside the description ?  is it allowed to have multiple lines ?

Reply:
Yes, multiple lines are allowed and it can be formatted nicely:
https://github.com/zephyrproject-rtos/zephyr/pull/25996/files#diff-ff68241e84846fc765b7a5d0e2c6bcc8

Reply:
done